[stable/kibana] Support custom labels on Service

NicolasT opened the pull request:
In line with various other charts, allow to set custom labels on the rendered `Service` object by setting values.

NicolasT committed:
[stable/kibana] Support custom labels on Service

NicolasT commented:
CC @compleatang

NicolasT commented:
CLA has been signed, I believe.

NicolasT commented:
CC @JessicaGreben 
/assign @viglesiasce

compleatang commented on stable/kibana/templates/service.yaml:
```diff
@@ -6,6 +6,9 @@ metadata:
     chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+    {{- range $key, $value := .Values.service.labels }}
```
in keeping with helm defaults, can you outdent this line and the `{{- end }} line two spaces please? in general this makes the chart more readable.

compleatang commented:
@NicolasT I added one minor change request. Also, can you please add a line in the README so that folks understand the functionality exists without having to parse the values.yaml file? 

Thanks!

NicolasT committed:
[stable/kibana] Process #5901 review comments

- Outdent template `if` and `end` markers
- Document new parameter in `README.md`

See: #5901
See: 857dcc3662ff4407039595bdaf692603152b4c3e
See: https://github.com/kubernetes/charts/pull/5901
See: https://github.com/kubernetes/charts/pull/5901#pullrequestreview-125719949
See: https://github.com/kubernetes/charts/pull/5901#issuecomment-394479332

compleatang commented:
LGTM

NicolasT commented:
Anything I can do to make this move forward?

unguiculus committed:
Bump minor version for new feature

unguiculus commented:
/ok-to-test

unguiculus commented:
Your branch is outdated. You need to merge in the current master so CI works.

NicolasT committed:
Merge remote-tracking branch 'kubernetes/master' into kibana-service-labels

* kubernetes/master: (411 commits)
  [stable/efs-provisioner] Chart for efs-provisioner, a Kubernetes external-storage provisioner (#3233)
  [stable/filebeat] filebeat fixes (#6332)
  [stable/stolon] Add support for priorityClasses (#6607)
  [stable/external-dns] Add support for priorityClasses (#6606)
  [stable/minio] Add support for priorityClasses (#6604)
  [stable/cluster-autoscaler] Add support for priorityClasses (#6603)
  [stable/oauth2-proxy] Add support for priorityClasses (#6586)
  [stable/elasticsearch-exporter] add support for priorityClasses (#6584)
  [stable/traefik] adding support for traefik wildcard certificates (#6015)
  gcloud-sqlproxy: add tolerations and affinity to the Deployment (#6495)
  Review readme (#6399)
  [stable/mongodb-replicaset] Prometheus Metrics export (#6282)
  [stable/artifactory-ha] Typo fix: livessProbed->livenessProbed (#6462)
  [stable/artifactory] livessProbed->livenessProbed (#6461)
  [incubator/kube-spot-termination-notice-handler] Add Support for Tolerations (#5813)
  [stable/kanister-operator] RBAC changes and kanister profile creation (#6280)
  fix redis-ha NOTE.txt, stable/redis-ha don't create master-0 pod (#6131)
  [stable/concourse] add support for custom envvars for the web containers (#6441)
  upgrade to latest prometheus release 2.3.2 and alertmanager 0.15.1 (#6623)
  cert-manager: fast-forward to upstream 777ce6f4 (#6625)
  ...

NicolasT committed:
Merge remote-tracking branch 'origin/kibana-service-labels' into kibana-service-labels

* origin/kibana-service-labels:
  Bump minor version for new feature

unguiculus committed:
Merge branch 'master' into kibana-service-labels
